app/models/BannerTests.scala

```diff
@@ -0,0 +1,64 @@
+package models
+
+import enumeratum.{CirceEnum, Enum, EnumEntry}
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.auto._
+import io.circe.{Decoder, Encoder}
+
+import scala.collection.immutable.IndexedSeq
+
+sealed trait OphanProduct extends EnumEntry
+
+object OphanProduct extends Enum[OphanProduct] with CirceEnum[OphanProduct] {
+  override val values: IndexedSeq[OphanProduct] = findValues
+
+  case object CONTRIBUTION extends Product
+  case object RECURRING_CONTRIBUTION extends Product
+  case object MEMBERSHIP_SUPPORTER extends Product
+  case object MEMBERSHIP_PATRON extends Product
+  case object MEMBERSHIP_PARTNER extends Product
+  case object DIGITAL_SUBSCRIPTION extends Product
+  case object PRINT_SUBSCRIPTION extends Product
+}
+
+case class BannerVariant(
```

Comment:
The model in frontend is a bit of a mess - lots of legacy fields.
Let's focus on the tool requirements for now:
```
BannerVariant(
  name: String,
  headline: Option[String],
  body: String,
  highlightedText: String,
  cta: Option[Cta],
  secondaryCta: Option[Cta],
  hasTicker: Boolean
)
```

Reply:
we'll do any necessary transformation on frontend, and in the long term change the model there

Reply:
Keep it simple. I like it.